docs/OPTIMIZATION_PURE_GPU_PIPELINE.md

```diff
@@ -51,3 +52,50 @@ So if a tensor's shape is `[1, height, width, 4]` or `[height, width, 4]`, the w
 downstream processing assumes the texture is an image, it doesn't need to
 do any additional coordination conversion. Each texel is mapped to a [CSS pixel](https://developer.mozilla.org/en-US/docs/Glossary/CSS_pixel) in an image. However, if the tensor has other shapes, downstream
 processing steps may need to reformat the data onto another texture. The `texShape` field has the `[height, width]` information of the texture, which can be used for transformation.
+
+# For webgpu backend
+The example below shows how to get the tensor buffer by calling
+`datatoGPU`. Also see this [example code](https://github.com/tensorflow/tfjs-examples/tree/master/gpu-pipeline/webgpu) for details.
```

Comment:
Should be "dataToGPU", and WebGL also has this typo.

Reply:
Done